tests/test_strutil.py

```diff
@@ -1,6 +1,6 @@
 import unittest
 
-from toolchest.strutil import list_or_splitstr
+from toolchest.strutil import list_or_splitstr, nym
 
 
 class test_list_or_splitstr(unittest.TestCase):
```

Comment:
suggestion: I realize this library has a number of older tests that still use unittest, but for new code, I would rather see us use pytest, like the rest of our projects (and newer code here).

Reply:
Done